### DISCUSSION TITLE
A lot of commits disappeared from my GitHub account.

### DISCUSSION THREAD

**Discussion by saeta-eth** [community/community#102264] [New to GitHub]
I've encountered an issue where my commit history in a particular project appears to have vanished, and I'm unsure of the underlying reasons. For instance, despite having contributed to these two open source projects, I don't see myself listed as a contributor:

- https://github.com/AelinXYZ/aelin-frontend-v2
- https://github.com/AelinXYZ/aelin-subgraph

Do you know how to fix this issue? Is there a way to link my commits again?

**Comment by SuperZekes** [comment]
Try logging in and out of your account this may fix it. If not use look into https://support.github.com/
